all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -23,25 +23,23 @@ items:
 - apiVersion: extensions/v1beta1
   kind: Deployment
   metadata:
-    name: jaeger-all-in-one
+    name: jaeger-deployment
```

Comment:
So when you do `kubectl whatever` you can find out that deployed thing is all-in-one and not production.